entries/s/singpass.gov.sg.json

```diff
@@ -4,15 +4,17 @@
     "url": "https://www.singpass.gov.sg/",
     "tfa": [
       "sms",
-      "custom-software",
```

Comment:
The documentation linked (Singpass 2FA > Singpass app 2FA) also says the following:
> With Singpass app, you can log in to government digital services using your biometrics (fingerprint or face) or 6-digit passcode.

Wouldn't this count as custom software?